app.py

```diff
@@ -41,20 +46,50 @@ def _benchmark_import_time() -> Dict[str, float]:
     }
 
 
-def _benchmar_flows() -> Dict[str, None]:
-    # TODO
+def _benchmark_avg_flow_time() -> Dict[str, float]:
+    """Benchmark on a simple flow operation.
+
+    Reurns:
+        A dict mapping of import time in seconds as float number.
+    """
     fs = [
-        Flow(),
         Flow().add(),
         Flow().add().add(),
-        Flow().add().add(needs='gateway')]
-
+        Flow().add().add().add(),
+        Flow().add().add().add(needs='gateway')
+    ]
+    st = time.perf_counter()
     for f in fs:
-        f.post('/', (Document() for _ in range(10000)))
-        f.post('/', (Document(blob=sp.coo_matrix([0, 0, 0, 1, 0])) for _ in range(10000)))
+        with f:
+            f.post(on='/', inputs=__doc_generator, request_size=10)
+    flow_time = time.perf_counter() - st
```

Comment:
This should be measured per flow, right?

Reply:
I am confused about the metrics. How can we express the metrics if we measure per flow? What's your suggestion?